Source/CarthageKit/Project.swift

```diff
@@ -283,7 +283,7 @@ public final class Project {
 			.on(next: { newVersions in
 				self.cachedVersions[project] = newVersions
 			})
-			.flatMap(.Concat) { versions in SignalProducer(values: versions) }
+			.flatMap(.Concat) { versions in SignalProducer<PinnedVersion, CarthageError>(values: versions) }
```

Comment:
We must disambiguate `init(values:)` here, because of the overload introduced in https://github.com/ReactiveCocoa/ReactiveCocoa/pull/2928. Without the type hint, the compiler can't infer the correct type, there are two possibilities:

- `SignalProducer<PinnedVersion, CarthageError>`
- `SignalProducer<[PinnedVersion], CarthageError>`